### PR TITLE
Dịch "alexnet_vn" - Phần 1

### LINKED ISSUE #1319: Dịch "alexnet_vn" - Phần 1
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_convolutional-modern/alexnet_vn.md#L1-L47

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by namndtth**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1319

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_convolutional-modern/alexnet_vn.md [hunk 1]
```diff
@@ -4,46 +4,42 @@
...
 Although LeNet achieved good results on early small datasets, the performance and feasability of training convolutional networks on larger, more realistic datasets had yet to be established.
 In fact, for much of the intervening time between the early 1990s and the watershed results of 2012, neural networks were often surpassed by other machine learning methods, such as support vector machines.
 -->
-
-*dịch đoạn phía trên*
-
-
+Mặc dù mạng nơ-ron tích chập nổi tiếng trong cộng đồng thị giác máy tính và học máy sau khi mạng LeNet được giới thiệu, chúng vẫn chưa thống trị lĩnh vực này ngay lập tức.
+Mặc dù LeNet đã đạt được những kết quả tốt trên những tập dữ liệu nhỏ, hiệu suất và tính khả thi của việc huấn luyện mạng tích chập trên một tập dữ liệu lớn hơn, thực tế hơn vẫn chưa được thiết lập.
+Trong thực tế, hầu hết khoảng thời gian giữa đầu những năm 1990 và những kết quả quan trọng trong năm 2012, mạng nơ-ron tích chập vẫn thường bị vượt qua bởi những phương pháp học máy khác, như SVM (support vector machines).
```
Comment (lkhphuc) [inline chapter_convolutional-modern/alexnet_vn.md:16]:
```suggestion
Trong thực tế, hầu hết khoảng thời gian giữa đầu những năm 1990 cho tới thành tựu to lớn năm 2012, mạng nơ-ron tích chập vẫn thường bị vượt qua bởi những phương pháp học máy khác, như là Máy Vector hỗ trợ - SVM.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 2]
```diff
@@ -4,46 +4,42 @@
...
 That is although the inputs to convolutional networks consist of raw or lightly-processed (e.g., by centering) pixel values, practitioners would never feed raw pixels into traditional models.
 Instead, typical computer vision pipelines consisted of manually engineering feature extraction pipelines.
 Rather than *learn the features*, the features were *crafted*.
 Most of the progress came from having more clever ideas for features, and the learning algorithm was often relegated to an afterthought.
 -->
-
-*dịch đoạn phía trên*
-
+Đối với thị giác máy tính, sự so sánh này có lẽ không công bằng.
+Lí do là giá trị đầu vào của mạng tích chập chỉ bao gồm giá trị điểm ảnh chưa được xử lý hoặc được xử lý nhưng không quá nhiều, trong khi đó những giá trị thô này không bao giờ được đưa vào các mô hình truyền thống để học.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/alexnet_vn.md:25]:
```suggestion
Lí do là giá trị đầu vào của mạng tích chập chỉ bao gồm giá trị điểm ảnh chưa được xử lý hoặc được xử lý nhẹ, trong khi đó những giá trị thô này không bao giờ được đưa trực tiếp vào các mô hình truyền thống để học.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 3]
```diff
@@ -4,46 +4,42 @@
...
 Although some neural network accelerators were available in the 1990s, they were not yet sufficiently powerful to make deep multichannel, 
 multilayer convolutional neural networks with a large number of parameters.
 Moreover, datasets were still relatively small.
 Added to these obstacles, key tricks for training neural networks including parameter initialization heuristics, clever variants of stochastic gradient descent,
 non-squashing activation functions, and effective regularization techniques were still missing.
 -->
-
-*dịch đoạn phía trên*
-
+Mặc dù đã có những động lực thúc đẩy mạng nơ-ron vào đầu những năm 1990, nhưng chúng chưa đủ mạnh để có thể tạo ra những mạng nơ-ron với nhiều kênh, nhiều tầng với số lượng tham số rất lớn.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/alexnet_vn.md:36]:
```suggestion
Mặc dù cũng đã có các thiết bị phần cứng tăng tốc độ thực thi mạng nơ-ron vào đầu những năm 1990, nhưng chúng vẫn chưa đủ mạnh để có thể tạo ra những mạng nơ-ron với nhiều kênh, nhiều tầng với số lượng tham số rất lớn.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 4]
```diff
@@ -4,46 +4,42 @@
...
+Ngoài ra, những tập dữ liệu liên quan vẫn còn rất nhỏ.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/alexnet_vn.md:37]:
```suggestion
Ngoài ra, những tập dữ liệu vẫn còn tương đối nhỏ.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 5]
```diff
@@ -4,46 +4,42 @@
...
+Thêm vào những trở ngại này, những thủ thuật chính để huấn luyện mạng nơ-ron bao gồm khởi tạo tham số dựa trên kinh nghiệm, các biến thể khác tốt hơn của hạ gradient ngẫu nhiên, hàm kích hoạt không nén, và kĩ thuật điều chuẩn hiệu quả vẫn còn thiếu.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/alexnet_vn.md:38]:
```suggestion
Thêm vào những trở ngại này, những thủ thuật chính để huấn luyện mạng nơ-ron bao gồm khởi tạo tham số dựa trên kinh nghiệm, các biến thể khác tốt hơn của hạ gradient ngẫu nhiên, hàm kích hoạt không ép, và kĩ thuật điều chuẩn hiệu quả vẫn còn thiếu.
```
squashing là `ép` theo glossary.

File: chapter_convolutional-modern/alexnet_vn.md [hunk 6]
```diff
@@ -4,46 +4,42 @@
...
 <!--
 Thus, rather than training *end-to-end* (pixel to classification) systems, classical pipelines looked more like this:
 -->
-
-*dịch đoạn phía trên*
-
+Vì vây, thay vì huấn luyện *đầu-cuối* (điểm ảnh đến phân loại) hệ thống, các pipeline cổ điển trông giống như thế này:
```
Comment (lkhphuc) [inline chapter_convolutional-modern/alexnet_vn.md:42]:
```suggestion
Vì vây, thay vì huấn luyện các hệ thống *đầu-cuối* (điểm ảnh đến phân loại), các pipeline cổ điển trông giống như thế này:
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 7]
```diff
@@ -4,46 +4,42 @@
...
 <!--
 Although convolutional neural networks were well known in the computer vision and machine learning communities following the introduction of LeNet, they did not immediately dominate the field.
 Although LeNet achieved good results on early small datasets, the performance and feasability of training convolutional networks on larger, more realistic datasets had yet to be established.
 In fact, for much of the intervening time between the early 1990s and the watershed results of 2012, neural networks were often surpassed by other machine learning methods, such as support vector machines.
 -->
-
-*dịch đoạn phía trên*
-
-
+Mặc dù mạng nơ-ron tích chập nổi tiếng trong cộng đồng thị giác máy tính và học máy sau khi mạng LeNet được giới thiệu, chúng vẫn chưa thống trị lĩnh vực này ngay lập tức.
```
Comment (minhduc0711) [inline chapter_convolutional-modern/alexnet_vn.md:14]:
```suggestion
Mặc dù mạng nơ-ron tích chập đã trở nên nổi tiếng trong cộng đồng thị giác máy tính và học máy sau khi mạng LeNet được giới thiệu, chúng vẫn chưa thống trị lĩnh vực này ngay lập tức.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 8]
```diff
@@ -4,46 +4,42 @@
...
+Mặc dù LeNet đã đạt được những kết quả tốt trên những tập dữ liệu nhỏ, hiệu suất và tính khả thi của việc huấn luyện mạng tích chập trên một tập dữ liệu lớn hơn, thực tế hơn vẫn chưa được thiết lập.
```
Comment (minhduc0711) [inline chapter_convolutional-modern/alexnet_vn.md:15]:
```suggestion
Mặc dù LeNet đã đạt được kết quả tốt trên những tập dữ liệu nhỏ, hiệu năng và tính khả thi của việc huấn luyện mạng tích chập trên một tập dữ liệu lớn và thực tế hơn vẫn chưa được thiết lập.
```

File: chapter_convolutional-modern/alexnet_vn.md (hunk 2 above)
Comment (minhduc0711) [reply chapter_convolutional-modern/alexnet_vn.md:25]: còn cụm `center pixels` nên dịch thế nào nhỉ

File: chapter_convolutional-modern/alexnet_vn.md [hunk 9]
```diff
@@ -4,46 +4,42 @@
...
 Instead, typical computer vision pipelines consisted of manually engineering feature extraction pipelines.
 Rather than *learn the features*, the features were *crafted*.
 Most of the progress came from having more clever ideas for features, and the learning algorithm was often relegated to an afterthought.
 -->
-
-*dịch đoạn phía trên*
-
+Đối với thị giác máy tính, sự so sánh này có lẽ không công bằng.
+Lí do là giá trị đầu vào của mạng tích chập chỉ bao gồm giá trị điểm ảnh chưa được xử lý hoặc được xử lý nhưng không quá nhiều, trong khi đó những giá trị thô này không bao giờ được đưa vào các mô hình truyền thống để học.
+Thay vào đó, những hệ thống thị giác máy tính sẽ có những pipeline dùng để rút trích những đặc trưng một cách thủ công.
```
Comment (minhduc0711) [inline chapter_convolutional-modern/alexnet_vn.md:26]:
```suggestion
Thay vào đó, những hệ thống thị giác máy tính sẽ có những pipeline dùng để trích xuất đặc trưng một cách thủ công.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 10]
```diff
@@ -4,46 +4,42 @@
...
+Thay vì *học đặc trưng*, những đặc trưng được *tạo ra*.
```
Comment (minhduc0711) [inline chapter_convolutional-modern/alexnet_vn.md:27]:
```suggestion
Thay vì *học đặc trưng*, những đặc trưng được *tạo thủ công*.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 11]
```diff
@@ -4,46 +4,42 @@
...
+Hầu hết những phát triển của các hệ thống đều đến từ việc có nhiều ý tưởng thông minh hơn trong việc tạo ra đặc trưng, và thuật toán dùng để học thường ít được để ý hơn.
```
Comment (minhduc0711) [inline chapter_convolutional-modern/alexnet_vn.md:28]:
```suggestion
Hầu hết những tiến triển đều đến từ các ý tưởng thông minh hơn trong việc tạo ra đặc trưng và thuật toán học thường ít được để ý hơn.
```

File: chapter_convolutional-modern/alexnet_vn.md (hunk 3 above)
Comment (minhduc0711) [reply chapter_convolutional-modern/alexnet_vn.md:36]: mình xóa từ `nhưng` trong suggestion của @lkhphuc 
```suggestion
Mặc dù cũng đã có các thiết bị phần cứng tăng tốc độ thực thi mạng nơ-ron vào đầu những năm 1990, chúng vẫn chưa đủ mạnh để có thể tạo ra những mạng nơ-ron với nhiều kênh, nhiều tầng với số lượng tham số rất lớn.
```

File: chapter_convolutional-modern/alexnet_vn.md [hunk 12]
```diff
@@ -4,46 +4,42 @@
 <!--
 # Deep Convolutional Neural Networks (AlexNet)
 -->
-
-# *dịch tiêu đề phía trên*
+Mạng Nơ-ron Tích Chập sâu (AlexNet)
```
Comment (lkhphuc) [inline chapter_convolutional-modern/alexnet_vn.md:7]:
```suggestion
# Mạng Nơ-ron Tích Chập sâu (AlexNet)
```
